boards/genericSTM32F407IGT6.json

```diff
@@ -0,0 +1,55 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F407xx -DSTM32F4",
+    "f_cpu": "168000000L",
+    "hwids": [
+      [
+        "0x1EAF",
+        "0x0003"
+      ],
+      [
+        "0x0483",
+        "0x3748"
+      ]
+    ],
+    "mcu": "stm32f407igt6",
+    "product_line": "STM32F407xx",
+    "variant": "Generic_F4x7Ix"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32F407IG",
+    "openocd_extra_args": [
+      "-c",
+      "reset_config none"
+    ],
+    "openocd_target": "stm32f4x",
+    "svd_path": "STM32F40x.svd"
+  },
+  "frameworks": [
+    "cmsis",
+    "stm32cube",
+    "libopencm3"
+  ],
+  "name": "STM32F407IG (192k RAM. 1024k Flash)",
+  "upload": {
+    "disable_flushing": false,
+    "maximum_ram_size": 196608,
+    "maximum_size": 1048576,
+    "protocol": "stlink",
+    "protocols": [
```

Comment:
Could you please extend the list of supported protocol as for example in this manifest https://github.com/platformio/platform-ststm32/blob/develop/boards/genericSTM32F446RE.json#L28-L32

Reply:
Fixed. Would it make sense to also change it in https://github.com/platformio/platform-ststm32/blob/develop/boards/genericSTM32F407VGT6.json then ?

Reply:
Sounds like a good idea, but seems out of the scope for this PR.